widgetssdk/src/main/java/com/glia/widgets/call/CallController.kt

```diff
@@ -229,12 +221,8 @@ internal class CallController(
         dialogController.dismissDialogs()
     }
 
-    private fun queueForEngagement(queueId: String?, visitorContextAssetId: String?, mediaType: Engagement.MediaType) {
-        disposable.add(
-            gliaQueueForMediaEngagementUseCase
-                .execute(queueId, visitorContextAssetId, mediaType)
-                .subscribe({ queueForEngagementStarted() }) { queueForEngagementError(it) }
-        )
+    private fun queueForEngagement(queueId: String, visitorContextAssetId: String?, mediaType: Engagement.MediaType) {
+        queueForEngagementUseCase(queueId, mediaType, visitorContextAssetId)
```

Comment:
Not critical. I would suggest to rename using some verb. For example, `startQueueForEngagement()`, `startQueueForEngagementUseCase()` because otherwise it's a bit unclear if it checks for queueing state or initiates it.